### DISCUSSION TITLE
Connect strapi project with existing postgres - postgis database

### DISCUSSION THREAD

**Discussion by elenaVal** [strapi/strapi#6885] [Legacy Topics]
Hello,

At first i would like to say that i am very new to the Strapi framework. 
Currently, i am trying to create a strapi project and connect it with an existing postgres database. This database has already some manually created tables. 

So, i tried to create the project with the existing database as well as i have made one modification in order to define the schema parameter in the database file as you can see below:
```
{
  "defaultConnection": "default",
  "connections": {
    "default": {
      "connector": "strapi-hook-bookshelf",
      "settings": {
        "client": "postgres",
        "host": "127.0.0.1",
        "port": 5432,
        "database": "testdb",
	"schema":"public",
        "username": "postgres",
        "password": "$$$$"
      },
      "options": {}
    }
  }
}
```
The project creation is finished without any errors and i was able to run the project, but in the strapi admin page, i cannot see any of the existing tables. 
Do you have any ideas why? Shall i follow different steps?
Thanks a lot.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
